neo-cli/Privatenet-Neo-Cli-Tutorial/set-privatenet-neo-cli.md

```diff
@@ -0,0 +1,111 @@
+## Setting a private net with neo-cli
+
+1. Clone the newer version of neo-node from [Neo-Node](https://github.com/neo-project/neo-node) for each node you desire to set in your privatenet and one that will receive a different configuration and will be used later.
```

Comment:
não entendi muito bem "and one that will receive a different configuration and will be used later"